### PR TITLE
Fix: rename UserAgent edge to msedge

### PR DESCRIPTION
**Pull request by robbinjanssen**
### What is the reason for this PR?
The useragent is called ['edge'](https://github.com/FakerPHP/Faker/blob/main/src/Faker/Provider/UserAgent.php#L7), when calling a random userAgent via [`userAgent()`](https://github.com/FakerPHP/Faker/blob/main/src/Faker/Provider/UserAgent.php#L57) it tries to call `userAgent->edge()` but the function is currently called [`msedge()`](https://github.com/FakerPHP/Faker/blob/main/src/Faker/Provider/UserAgent.php#L91). This PR renames that static `edge` into `msedge`.

- [ ] A new feature
- [X] Fixed an issue

### Author's checklist

- [x] Follow the [Contribution Guide](https://github.com/FakerPHP/Faker/blob/main/.github/CONTRIBUTING.md)
- [ ] New features and changes are [documented](https://github.com/FakerPHP/fakerphp.github.io)

### Summary of changes

tl;dr - rename `edge` to `msedge`

### Review checklist

- [ ] All checks have passed
- [ ] Changes are approved by maintainer

### CODE REVIEW

**Review by pimjansen — APPROVED** [review]

**Comment by pimjansen** [comment]
LGTM

**Comment by pimjansen** [comment]
@robbinjanssen see CS issue in build

**Comment by robbinjanssen** [comment]
@pimjansen applied the CS changes 👍
